Simplified/Reader2/LCP/LCPLibraryService.swift

```diff
@@ -42,81 +66,23 @@ import ReadiumLCP
   ///   - downloadTask: `URLSessionDownloadTask` that downloaded the publication.
   ///   - error: `NSError` if any.
   @objc func fulfill(_ file: URL, completion: @escaping (_ localUrl: URL?, _ error: NSError?) -> Void) {
-    lcpService.importPublication(from: file, authentication: self) { result, error in
-      var nsError: NSError?
-      if let error = error {
+    self.lcpService.acquirePublication(from: file) { result in
+      do {
+        let publication = try result.get()
+        completion(publication.localURL, nil)
+      } catch {
         let domain = "LCP fulfillment error"
         let code = 0
```

Comment:
can we use an NYPLErrorCode instead of 0?

Reply:
yes, added `lcpDRMFulfillmentFail`